Fixing Issues #52 & #53 - Disabling

TheWitness opened the pull request.

TheWitness committed:
Fixing Issues #52 & #53 - Disabling

evilnate83 commented:
I replaced the contents of the file with the patch and the map editor loads now.

TheWitness committed:
Fixing #57 - Wrong table alias

Invalid argument supplied for foreach() in file: /lib/ds-common.php on line: 66

TheWitness committed:
Additional fix for #57 - missing check

TheWitness committed:
Fixing Issue #55 - Bad variable

TheWitness committed:
Fixing Issue #50 - Link order

Switching Left for Right on the selection.

TheWitness committed:
Fixing trimming of file paths in #58

TheWitness committed:
mogrify images
